### ISSUE TITLE
Comically large body in Vivecraft

### ISSUE THREAD

**Issue by RandomVR** [tr7zw/FirstPersonModel#328]
The actual body renders properly in VR from what I can see, (including arm animations) but for some reason it is extremely large (shown in attached image)
 ##
![OculusScreenshot1674844143](https://user-images.githubusercontent.com/104455920/215167160-a599e720-fa4b-4e8e-ab00-45a191cae1cd.jpeg)

**Comment by tr7zw** [comment]
I mean, vivecraft was never said to be supported, and would need a ton of work to have the hands right. No clue why it would be this big.

**Comment by RandomVR** [comment]
> I mean, vivecraft was never said to be supported, and would need a ton of work to have the hands right. No clue why it would be this big.

Yeah I didn't go in expecting it to work, but this is the last thing I expected to happen. Just thought it would be worth reporting anyway

**Comment by tr7zw** [comment]
At least with MCXR it mostly worked, so kinda expected the same. No clue if/when I get around to look into it.

**Comment by RandomVR** [comment]
Yeah, thanks for the quick response anyway
